### PR TITLE
FIxed Overlapping and scrollbar issues

### PR DESCRIPTION
**Pull request by adebayokehinde**
This PR addresses the issue https://github.com/capofficial/client/issues/37 

image

Overlapping was caused by the grid's CSS setting missing an overflow configuration

I fixed this issue by adding overflow to the grid and **@media(max-width:600px) grid{}** and also the persistent scrollbar is now gone 

image

I'm willing to update according to your preferences :) @kappacappa

### CODE REVIEW

File: src/global.css [hunk 1]
```diff
@@ -57,12 +63,36 @@ button{
...
+}
+
+@media all and (min-width: 769px) {
+  body {
+    overflow: hidden;
+  }
+
+}
+
+.grid {
```
Comment (kappacappa) [inline src/global.css:86]: This should probably be contained in Trade.svelte, not in global CSS.

**Comment by kappacappa** [comment]
@adebayokehinde thanks, added comment.

File: src/global.css (hunk 1 above)
Comment (adebayokehinde) [reply src/global.css:86]: Done

Comment (kappacappa) [reply src/global.css:86]: I don't see any new commits?

Comment (adebayokehinde) [reply src/global.css]: > I don't see any new commits.

My apologies, I've pushed it now

**Comment by kappacappa** [comment]
@adebayokehinde Thanks, but there is unknown code that has been introduced here: https://github.com/capofficial/client/pull/45/files#diff-b6208c7a9db5387aed5b0cc81577feddf2ca323d7e28bc98c55e2b4fd9b3f808R21

I appreciate your contributions, but I believe the code quality is not sufficient to be eligible for bounties. We're looking for experienced developers who can deliver high quality, tested code from the start.

I encourage you to keep contributing and I'd be happy to look over your future changes as time permits.

**Comment by adebayokehinde** [comment]
> @adebayokehinde Thanks, but there is unknown code that has been introduced here: https://github.com/capofficial/client/pull/45/files#diff-b6208c7a9db5387aed5b0cc81577feddf2ca323d7e28bc98c55e2b4fd9b3f808R21
> 
> I appreciate your contributions, but I believe the code quality is not sufficient to be eligible for bounties. We're looking for experienced developers who can deliver high quality, tested code from the start.
> 
> I encourage you to keep contributing and I'd be happy to look over your future changes as time permits.

Yes, it's actually work in progress for the next issue, I mistakingly pushed it along side the third commit.

**Comment by adebayokehinde** [comment]
I removed the unused code :)

**Comment by kappacappa** [comment]
@adebayokehinde as I understand the change is simply to add the update the overflow on the grid in https://github.com/capofficial/client/pull/45/files#diff-f1e97e15940e636ecad4710211e6c199334c75d3f29968b579c39129bd557378

If so, can you please discard any changes to other files as they would not make sense as part of this commit.

**Comment by adebayokehinde** [comment]
> @adebayokehinde as I understand the change is simply to add the update the overflow on the grid in https://github.com/capofficial/client/pull/45/files#diff-f1e97e15940e636ecad4710211e6c199334c75d3f29968b579c39129bd557378
> 
> If so, can you please discard any changes to other files as they would not make sense as part of this commit.

Done, kindly review

**Comment by kappacappa** [comment]
@adebayokehinde thanks, can you please confirm it is working as expected on your local machine, with different window sizes, or in mobile device simulators? If so, I'll merge.

**Comment by adebayokehinde** [comment]
> @adebayokehinde thanks, can you please confirm it is working as expected on your local machine, with different window sizes, or in mobile device simulators? If so, I'll merge.

Yes it works well on my local machine, I tested on all mobile device simulators
On desktop, it works fine even down to 600px in width, no overlapping

**Pull request merged**